### ISSUE TITLE
For those who did not sit down to install .Net 6.0 command   Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore  will not install.

### ISSUE THREAD

**Issue by AlexFromKiyv** [dotnet/AspNetCore.Docs#24271]
For those who did not sit down to install .Net 6.0 commands:

  Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
  Install-Package Microsoft.EntityFrameworkCore.SqlServer

will not end in success.

In order to go further, you need to install the previous version (I currently have 5.0.12)

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ad06677-5842-c1aa-153f-7993a9660dfa
* Version Independent ID: 8daf79df-eac4-358b-e6f0-6def45bd54da
* Content: [Tutorial: Get started with EF Core in an ASP.NET MVC web app](https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/intro?view=aspnetcore-5.0#set-up-the-site-style)
* Content Source: [aspnetcore/data/ef-mvc/intro.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/data/ef-mvc/intro.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
> In order to go further, you need to install the previous version (I currently have 5.0.12)

You mean ***uninstall***

**Comment by AlexFromKiyv** [comment]
The system requirements are listed at the beginning of the article. 
Visual Studio 2019 16.8 or later 
.NET 5.0 SDK

I got to the code and ran the command

Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore

the package did not install and required .Net 6.0

I wanted to quickly recall the process of setting up EF, and I couldn't move on with that.
The first thought was to install Visual Studio 2022 on a separate computer. 
I remembered that Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore version 5.0 can be installed through the package manager. And it helped me quickly remember how to work with EF.

It would be nice if the article would indicate what to do for the one whose projects have not yet been made on .Net 6.0
